### DISCUSSION TITLE
Seeking Advice: Connecting an SDR UE to srsRAN gNB/eNB in TDD Mode

### DISCUSSION THREAD

**Discussion by Tianyi94** [srsran/srsRAN_Project#51] [General Help]
Hi folks,

I am trying to use an SDR as UE in TDD mode. I tried to connect the srsUE from srsRAN_4G to srsenb SA mode from srsRAN_4G and srsgnb from srsRAN_project but failed. I know the problem is the SCS but I am still wondering whether anyone has any successful experience connecting the UE SDR to any of the srsRAN gNB/eNB?     

Hello everyone,

I'm currently attempting to use a SDR as UE in TDD mode. Specifically, I've been trying to establish a connection between the srsUE in srsRAN_4G and the SA mode srsenb in srsRAN_4G, as well as the srsgnb in srsRAN_project, but unfortunately, my attempts have been unsuccessful so far.

While I understand that the problem is from the SCS, I was wondering if anyone has had success connecting an SDR UE to either the srsRAN gNB or eNB, and if so, if they could offer any advice or guidance.

Thank you!

Best,
Tianyi

**Comment by brendan-mcauliffe** [comment]
Hi Tianyi, we have a new tutorial showing how to connect srsUE to the srsRAN Project gNB. You can find it [here](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html). Just remember to download the latest version of both srsRAN 4G and srsRAN Project before you start!
